hippo4j-spring-boot/hippo4j-spring-boot-starter/src/main/java/cn/hippo4j/springboot/starter/core/ClientWorker.java

```diff
@@ -44,19 +44,7 @@
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
 
-import static cn.hippo4j.common.constant.Constants.CLIENT_VERSION;
-import static cn.hippo4j.common.constant.Constants.CONFIG_CONTROLLER_PATH;
-import static cn.hippo4j.common.constant.Constants.CONFIG_LONG_POLL_TIMEOUT;
-import static cn.hippo4j.common.constant.Constants.GROUP_KEY_DELIMITER_TRANSLATION;
-import static cn.hippo4j.common.constant.Constants.LINE_SEPARATOR;
-import static cn.hippo4j.common.constant.Constants.LISTENER_PATH;
-import static cn.hippo4j.common.constant.Constants.LONG_PULLING_CLIENT_IDENTIFICATION;
-import static cn.hippo4j.common.constant.Constants.LONG_PULLING_TIMEOUT;
-import static cn.hippo4j.common.constant.Constants.LONG_PULLING_TIMEOUT_NO_HANGUP;
-import static cn.hippo4j.common.constant.Constants.NULL;
-import static cn.hippo4j.common.constant.Constants.PROBE_MODIFY_REQUEST;
-import static cn.hippo4j.common.constant.Constants.WEIGHT_CONFIGS;
-import static cn.hippo4j.common.constant.Constants.WORD_SEPARATOR;
+import static cn.hippo4j.common.constant.Constants.*;
```

Comment:
Import * is not allowed

Reply:
I have fixed the "import *" issue in three files.